controllers/databasecluster_controller.go

```diff
@@ -1894,6 +1909,12 @@ func (r *DatabaseClusterReconciler) genPGDataSourceSpec(ctx context.Context, dat
 				"--type=immediate",
 				"--set=" + backupBaseName,
 			},
+			Resources: corev1.ResourceRequirements{
+				Limits: corev1.ResourceList{
+					corev1.ResourceMemory: resource.MustParse("128Mi"),
+					corev1.ResourceCPU:    resource.MustParse("200m"),
+				},
+			},
```

Comment:
will this part stay when the templates available? if not, let's have a comment here also about removing it. And for the limits too if relevant.